### DISCUSSION TITLE
How to use cash

### DISCUSSION THREAD

**Discussion by kerlama** [breezefront/community#19] [1. Help]
H

I have a question for you. We are using our custom catalog_produt_view.xml which calls form.phtml from our custom module.

In that phtml file, there is some basic jquery dom manipulation and events, which are now not working.

This part of code is not working, throwing: Uncaught ReferenceError: $ is not defined

$('.some-class').on('click', function() {});

Its in <script>, but if we add <script data-breeze> its not working either.

We also tried with breeze_catalog_product_view.xml and then we have, along with above one, these: https://i.imgur.com/CL240NV.png

If we use approach from "Migrate inline scripts" section from your documentation, then we get that require is not defined. 

So the question is, how to use cash library in phtml files, not just cash but all globals ?

**Comment by vovayatsyuk** [comment]
The correct way to write inline scripts is to use `require`:

```js
<script data-breeze>
require(['jquery'], ($) => {
    $('.some-class').on('click', function() {});
    // Underscore and other globals are available here too.
});
</script>
```

"Require is not defined" error means that you are using old breeze version. Use `composer outdated "swissup/*"` command to see if everything is up to date.

**Comment by kerlama** [comment]
Hi, thank you for reply.

Here's the output:
Direct dependencies required in composer.json:
swissup/ajaxlayerednavigation        1.3.59  1.3.60 Advanced catalog filters with ajax and seo support
swissup/breeze-blank                 1.2.2   1.2.3  Breeze Blank Theme

Transitive dependencies not required in composer.json:
swissup/module-ajaxlayerednavigation 1.3.59  1.3.60 Advanced catalog filters with ajax and seo support
swissup/module-breeze                1.13.10 1.14.4 Performant Magento Frontend
swissup/theme-frontend-breeze-blank  1.2.2   1.2.3  Clean, lightning-fast breeze-powered theme.

So I guess, upgrade is required.

Thanks again.
